### DISCUSSION TITLE
Mastodon (or any FediVerse) used as comments

### DISCUSSION THREAD

**Discussion by silverhook** [getpelican/pelican#3166] [Ideas]
I’m thinking of outsourcing comments to Mastodon (and therefore the whole Fediverse really).

Has anyone tried that yet?

My immediate thoughts would be to simply use:

1. https://github.com/pelican-plugins/fediverse to have all the posts tooted via my account
2. create an `iframe` to embed the thread (using the link to the toot)

I’m very open to better ideas :]

**Comment by ghost** [comment]
This article describes a way to add comments from Mastodon to a static site generated with Hugo with JavaScript: [Adding comments to your static blog with Mastodon](https://carlschwan.eu/2020/12/29/adding-comments-to-your-static-blog-with-mastodon/). It should be easy to adopt it to Pelican, though one would probably want to combine it with the functionality of https://github.com/pelican-plugins/fediverse so that the Mastodon posts are generated automatically.

**Comment by silverhook** [comment]
This is another option: https://grahammacphee.com/writing/mastodon-blog-comments

But one thing where I’m feeling a bit cautious is the privacy (esp. right to forget) aspect. If someone deleted their comment on the Fediverse (e.g. it expired), I want to respect that and not have a copy.
